### DISCUSSION TITLE
시간입력을 input? select box?

### DISCUSSION THREAD

**Discussion by codeisneverodd** [prgrms-fe-devcourse/FEDC2_TimeDivider_Root#22] [0. 개발]
시간 입력 부분을 어떤식으로 처리하는 것이 좋을까요

**Comment by codeisneverodd** [comment]
select box가 사용자 입장에서도 더 편할 것 같습니다.

**Comment by dahye1013** [comment]
일반 input=text는 유저가 어떤값을 입력해야하는지 인지하기가 어려울것 같아요! 
저도 select box 한표!
